lib/tasks/licenses.rake

```diff
@@ -39,9 +39,9 @@ namespace :licenses do  # rubocop:disable Metrics/BlockLength
 
   def cli
     @cli ||= begin
-               require 'license_finder'
-               LicenseFinder::CLI::Main.new
-             end
+      require 'license_finder'
+      LicenseFinder::CLI::Main.new
+    end
```

Comment:
My automatic indentation made it like before. I like more what you have here. But we should enforce single style somehow.

Reply:
Rubocop again, changed it:
```
lib/tasks/licenses.rake:44:14: W: [Correctable] Layout/BeginEndAlignment: end at 44, 13 is not aligned with @cli ||= begin at 41, 4.
             end
             ^^^
```

Reply:
Ok, I'll check if it is something local for me. I'm fine with the change.